src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DefaultVisualStudioRazorParser.cs

```diff
@@ -464,6 +464,7 @@ internal void OnDocumentStructureChanged(object state)
 
         private void ConfigureProjectEngine(RazorProjectEngineBuilder builder)
         {
+            builder.SetRootNamespace(_documentTracker.ProjectSnapshot.RootNamespace);
```

Comment:
It's possible the snapshot is null depending on how quick everything instantiates. You'll need to handle the situation when you can't resolve the root namespace here. For instance, in the TagHelper case below the document tracker just returns `null` where as this will null ref

Reply:
Also note, if the snapshot is `null`, when it populates everything will be reconfigured so there'd only be a slight lapse in a user having an unconfigured root namespace.